app/dashboard/models.py

```diff
@@ -1318,22 +1318,22 @@ def post_save_bounty(sender, instance, created, **kwargs):
         instance.featuring_date = timezone.now()
         instance.hyper_next_publication = timezone.now()
 
-        # Publish and pin on townsaquare
-        profile = Profile.objects.filter(handle='owocki').first()
-        metadata = {
-                'title': title,
-                'description': truncatechars(instance.issue_description_text, 500),
-                'url': instance.get_absolute_url(),
-                'ask': '#announce'
-        }
-        activity = Activity.objects.create(profile=profile, activity_type='hypercharge_bounty',
-                                           metadata=metadata, bounty=instance)
+        # Publish and pin on townsquare
+        profile = Profile.objects.filter(handle='connoroday').first()
```

Comment:
Can we do a followup PR that allows this to be set in the DB OR via the app environment settings?

Reply:
i moved this to settings as env variable